### PR TITLE
Google Calender docs

### PR DESCRIPTION
**Pull request by Shraddha2104**
✨ Added Google Calender docs.
Let me know if any changes are needed!

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+permalink: /nodes/n8n-nodes-base.googleCalender
+---
+
+# Google Calender
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: `Calendar`

Comment (ivov) [reply docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Same for all other instances below.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+- Event
+    - Add an event to the calender
+    - Delete an event
+    - Get an event
+	- Retrieve all the events
+    - Update an event
+
+
+
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Excess whitespace?

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Example Usage
+
+This workflow allows you to add an event to Google Calender. You can also find the [workflow](https://n8n.io/workflows/427) on the website. This example usage workflow would use the following two nodes.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: `would use` → `uses`

Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Maybe `allows you` → `shows you`?

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+![A workflow with the Google Calender node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Google Calender node
+
+1. First of all, you'll have to enter credentials for the Google Calender node. You can find out how to do that [here](../../../credentials/Google/README.md).
+2. To add a event to the calender, select the Event option under the *Resource* field.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: `an event`

Comment (ivov) [reply docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: I think (not sure) it should be `*Event option*`. Please double check.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+3. Select the Create option in the *Operation* field.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Italics for `Create option`?

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+4. Select the *Calender* from the dropdown list of the user's calender list.
+5. Enter your start date of your event in the *Start* field.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: `the start date`

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+6. Enter your end date of your event in the *End* field.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: `the end date`

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+7. Enable or disable *Use Default Reminder* field according to your needs.
+8. If you want to enter optional fields such as the attendees or location, etc as JSON data, then add those JSON parameters under the *Additional fields* field.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Optional fields should not be included in the instructions.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+9. Click on *Execute Node* to run the workflow.
+
+
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Extra whitespace?

**Review by ivov — CHANGES_REQUESTED** [review]
Looks good! Just small details.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md (hunk 3 above)
Comment (Shraddha2104) [reply docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: your version is better, but the reference docs has `allows you`

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+---
+permalink: /nodes/n8n-nodes-base.googleCalender
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Calendar is misspelled in the document and in the folder name.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+# Google Calender
+
+[Google Calender](https://www.google.com/calendar/) a time-management and scheduling calendar service developed by Google.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: A minor grammatical error, missing 'is'.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Google/README.md).
+:::
+
+## Basic Operations
+
+- Event
+    - Add an event to the calender
+    - Delete an event
+    - Get an event
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: Could you please make sure that the operations occur in the same order as in the node. Also, the description should be the same as the text in the node.

Screenshot 2020-06-29 at 09 04 43

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+![A workflow with the Google Calender node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Google Calender node
+
+1. First of all, you'll have to enter credentials for the Google Calender node. You can find out how to do that [here](../../../credentials/Google/README.md).
+2. To add an event to the calender, select *Event* under the *Resource* field.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: This and the next step don't need to be mentioned as they are the default settings on the node.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+3. Select the *Create* option in the *Operation* field.
+4. Select the *Calender* from the dropdown list of the user's calender list.
+5. Enter the start date of your event in the *Start* field.
+6. Enter the end date of your event in the *End* field.
+7. Enable or disable *Use Default Reminder* field according to your needs.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: We can skip the optional sections in the docs where there isn't a concrete instruction for the user to follow.

File: docs/nodes/nodes-library/nodes/Google/Google Calender/README.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+8. If you want to enter optional fields such as the attendees or location, etc as JSON data, then add those JSON parameters under the *Additional fields* field.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Google/Google Calender/README.md]: We can skip the optional sections in the docs where there isn't a concrete instruction for the user to follow.

**Review by tanay1337 — CHANGES_REQUESTED** [review]

**Comment by Shraddha2104** [comment]
Fixed docs according to @tanay1337's review !

**Comment by Shraddha2104** [comment]
Submitted to maintainers.

**Pull request closed without merging**